tests/components/roon/test_event.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for the Roon event entity."""
+
+
+from unittest.mock import MagicMock
+
+from homeassistant.components.event import ATTR_EVENT_TYPE, EventDeviceClass
+from homeassistant.components.roon.event import RoonEventEntity
+
+
+async def test_event() -> None:
+    """Test the roon event entity."""
+    event = RoonEventEntity("server", "test")
```

Comment:
Don't create the entity directly. Set up the integration while patching the library and assert the entity state in the state machine.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations